app/views/second_level_browse_page/_links.html.erb

```diff
@@ -35,13 +41,19 @@
   <% if page.related_topics.any? %>
     <div class="detailed-guidance">
       <h2>Detailed guidance</h2>
-      <ul>
+      <ul data-analytics-ecommerce
+          data-search-query
+          data-ecommerce-start-index=<%= "#{ecommerce_page_row_count}" %>
```

Comment:
```suggestion
          data-ecommerce-start-index=<%= "#{page.lists.sum(&:size)}" %>
```
and remove the `ecommerce_page_row_count`?

Reply:
unfortunately that won't work, because it's not the sum of the lists, there's these list.contents bits too.

So i could do something like `page.lists.sum{|list| list.contents.sum(&:size)}`... (<- i think that might work?)
But i figure putting a variable within the <li> that I'm trying to count would be stupider but perhaps easier to grok if you were browisng the code? 

What'd ya reckon?

Reply:
Hmm, yeah not sure. Either way it seems like logic that should be placed in a view. Would a helper function with a descriptive function name be more ideal?